modules/aws/aws.go

```diff
@@ -0,0 +1,2 @@
+// Package aws allows to interact with resources on Amazon Web Services.
+package aws
```

Comment:
What does this empty file do? I noticed there's a bunch of these and it's not obvious to me what purpose they serve.

Reply:
They are simply there to document the package. I think it's advantageous if the file containing the comment always has the same name as the package itself (and the directory it's contained in) so you don't have to search which file contains the package comment.